### PR TITLE
Vesting adjustments

### PR DESCRIPTION
**Pull request by JExtor**

### CODE REVIEW

File: contracts/vesting/Genesis.sol [hunk 1]
```diff
@@ -150,11 +150,11 @@ contract GenesisClaim is Ownable {
      * @param _address address
      * @return uint256
      */
-    function redeemableFor(address _address) public view returns (uint256) {
-        Term memory info = terms[_address];
-        uint256 max = circulatingSupply().mul(info.percent).mul(1e3);
+    function redeemableFor( address _address ) public view returns (uint) {
```
Comment (lolchocotaco) [inline contracts/vesting/Genesis.sol:153]: Why remove the explicit `uint256`

Comment (JExtor) [reply contracts/vesting/Genesis.sol:153]: Left out by accident. Did not not notice removed in the commit.

**Review by lolchocotaco — APPROVED** [review]
decimals are fun.

**Comment by lolchocotaco** [comment]
Address of deployed: 
Genesis: 0x5b2303Cc2fdE10fc8dE92d6728D0391b5dbaD9e4 
Investor: 0xF0fe16f47BA0808D54BBf56bb35A24C988e81260

**Pull request merged**